python/paddle/fluid/layers/control_flow.py

```diff
@@ -1798,6 +1800,9 @@ def complete(self):
                     intermediate.add(out_var_name)
         input_set = set([ipt.name for ipt in self.inputs])
 
+        # Todo(liym27) Here assume that all params are in recursive parent block
+        # but when minimize() called in control flow, some params may be in
```

Comment:
Question but nothing to change: in my understanding, params here are just forward input vars? So I think it won't be at conditional grad block?

Reply:
Got you.